examples/minimal/packages/client-react/src/mud/setupNetwork.ts

```diff
@@ -28,6 +29,15 @@ export async function setupNetwork() {
     account: burnerAccount,
   });
 
+  const write$ = new Subject<ContractWrite>();
+  const worldContract = createContract({
+    address: networkConfig.worldAddress as Hex,
+    abi: IWorld__factory.abi,
+    publicClient,
+    walletClient: burnerWalletClient,
+    onWrite: (write) => write$.next(write),
```

Comment:
wondering if we wanna export something like `write$` from `createContract` rather than the `onWrite` callback where user land has to convert it to an observable

downside is that common will need rxjs as a dep, which is quite heavy

Reply:
i like the lightweight `onWrite` callback